package.json

```diff
@@ -49,7 +49,7 @@
     "mongodb": "3.1.8",
     "mongoose": "5.3.6",
     "mu2": "~0.5.20",
-    "node-uuid": "~1.4.1",
+    "uuid": "^3.3.2",
```

Comment:
Let's use ~ instead of ^.

In addition, please add the change to CNR.

Reply:
Fixed with 2121f6bcf64e72ff8d52cd4394d2ece4705a18bd